server/text_generation_server/layers/attention/rocm.py

```diff
@@ -1,96 +1,169 @@
 import os
 import torch
-
+from text_generation_server.utils.import_utils import SYSTEM
 from loguru import logger
-import math
 
-from text_generation_server.utils.import_utils import SYSTEM
+major, minor = torch.cuda.get_device_capability()
+is_sm75 = major == 7 and minor == 5
+_PARTITION_SIZE = 512
 
-if SYSTEM != "xpu":
-    from text_generation_server.utils.flash_attn_triton import triton_attention
+use_triton = os.getenv("ROCM_USE_FLASH_ATTN_V2_TRITON", "").lower() in {"true", "1"}
+ENGINE = "triton" if use_triton else "ck"
+from .flash_attn_triton import triton_attention
```

Comment:
Does it make sense to import this when `use_triton == False`?

Reply:
Not really. I'll move it behind itś flag.